### PR TITLE
Player

### PR DESCRIPTION
**Pull request by saifjilani**
### Problem

Needed a points table which acts as a join table between game and player.
### Solution

Create a points table. 

Note: I probably should have split this into 3 different PR's. I added tests for player, game and points relationships to this PR as well. I also added the `pry-byebug` gem for debugging.

@muneeb1993

### CODE REVIEW

**Comment by muneebaahmad** [comment]
LGTN

**Comment by muneebaahmad** [comment]
Don't merge just yet! I want to look over it.

File: app/models/point.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Point < ApplicationRecord
```
Comment (muneebaahmad) [inline app/models/point.rb:1]: Whats the difference between point and points?

Comment (saifjilani) [reply app/models/point.rb:1]: It's just a Rails convention thing. Here's a good explanation: 

`Ruby on Rails follow linguistic convention. That means a model represents a single user, whereas a database table consists of many users.`

http://stackoverflow.com/questions/10078139/ruby-on-rails-plural-controller-and-singular-model-convention-explanation

Naming conventions in rails: http://itsignals.cascadia.com.au/?p=7

**Pull request merged**